BENCHMARKING.md

```diff
@@ -107,7 +113,80 @@ for i in $(seq 0 $(ssget -n)); do
 done
 ```
 
-### 3: Benchmarking overview
+### 3: Running benchmarks manually
+When compiling `ginkgo` with the flag `-DGINKGO_BUILD_BENCHMARKS=on`, a suite of
```

Comment:
nit: I would spell it Ginkgo and capitalize `ON` (we do the same in the beginning of the document).
```suggestion
When compiling Ginkgo with the flag `-DGINKGO_BUILD_BENCHMARKS=ON`, a suite of
```